pallets/identity/src/lib.rs

```diff
@@ -72,7 +72,10 @@ pub mod benchmarking;
 use pallet_identity_rpc_runtime_api::DidRecords as RpcDidRecords;
 use pallet_transaction_payment::{CddAndFeeDetails, ChargeTxFee};
 use polymesh_common_utilities::{
-    constants::did::{CDD_PROVIDERS_ID, GOVERNANCE_COMMITTEE_ID, SECURITY_TOKEN, USER},
+    constants::{
+        did::{CDD_PROVIDERS_ID, GOVERNANCE_COMMITTEE_ID, SECURITY_TOKEN, USER},
+        TREASURY_MODULE_ID,
```

Comment:
For my education, why is TREASURY_MODULE_ID referenced under did as well?

Reply:
The `TREASURY_MODULE_ID` under did is different from the one outside. The one outside is the actual `TREASURY_MODULE_ID` and the one inside is actually `TREASURY_MODULE_DID`. However, I used `TREASURY_MODULE_ID` to keep it consistant with `GOVERNANCE_COMMITTEE_ID` and `CDD_PROVIDERS_ID` which are also dids.

The real `TREASURY_MODULE_ID` (outside one) is an 8 bytes identifier while the other one is a did representation. Looking back at this, it might be best to rename the did constants.

Reply:
Renamed the did constants.